tests/voq/test_fabric_cli_and_db.py

```diff
@@ -0,0 +1,293 @@
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import random
+logger = logging.getLogger(__name__)
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
```

Comment:
t2 topology could be for Voq and Ethernet packet chassis. Can we make sure to run this test only voq chassis